### PR TITLE
reset button in NcWmsPanel works

### PR DESCRIPTION
**Pull request by danfruehauf**
* reset to original extent when clicked on reset

### CODE REVIEW

**Comment by danfruehauf** [comment]
@dnahodil This should work.

As a side note I'll just that this has to be rewritten to use the new filter architecture.

File: web-app/js/portal/details/NcWmsPanel.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
     PENDING_EVENT_ATTR: 'PENDING_EVENT',
 
     constructor: function(cfg) {
-
```
Comment (dnahodil) [inline web-app/js/portal/details/NcWmsPanel.js]: @danfruehauf 'developer discretion' means that you can choose how you want to write your own code (or code that you update). It *doesn't* mean you get to reformat every file you touch. What a huge waste of time that would be for everyone.

I mentioned this specific issue when we spoke about blank lines in code:
> The thing with developer discretion is we don't want one developer adding blank lines, and another removing them, and then someone else adding them back in.

If you feel strongly about this *please* raise it again in internal discussions and we can come to an agreement (I think the discussion will go in favour of no blank lines so you don't need to worry). Until it is sorted please add these blank lines back in for the purposes of this PR.

**Comment by dnahodil** [comment]
This change has fixed the issue we were seeing but now the original issue is back (the 'clear subset' button doesn't work).

File: web-app/js/portal/details/NcWmsPanel.js (hunk 1 above)
Comment (dnahodil) [inline web-app/js/portal/details/NcWmsPanel.js]: As a note I wouldn't let anyone apply a 'developer discretion'  code style like this (to a whole file even though they only touched a small part of it) even if the new code style was my preference, too. The point is not to reformat back-and-forth and it's not to apply your style to other people's work. If we've agreed on 'developer discretion' it means we all need to be happy with either.

Comment (danfruehauf) [inline web-app/js/portal/details/NcWmsPanel.js]: So if I change a function name, can I also get rid of the new line below it?

Comment (danfruehauf) [inline web-app/js/portal/details/NcWmsPanel.js]: From now on BTW, my 'developer discretion' will be to have 10 empty lines at the end of every function implementation.

**Pull request closed without merging**